subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -782,10 +781,8 @@ int ull_conn_llcp(struct ll_conn *conn, u32_t ticks_at_expire, u16_t lazy)
 
 		/* TODO: Optimize the checks below, maybe have common flag */
 
-		if (0) {
```

Comment:
This if (0) is related to the TODO above. Maybe you should not remove them until you fix the TODO

Reply:
> This if (0) is related to the TODO above. Maybe you should not remove them until you fix the TODO

@cvinayak ^^

Reply:
I don't think it is related to the TODO. The `if (0)` thing is used many places other than this.

Reply:
no, it is not related to TODO. Its safe to remove. It only made sense when the very first if clause too is conditionally compile-able. Here it is not the case.